src/ProjectNeon/Assets/Features/Battle/Cards/EnemyCardSelection.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyCardSelection : MonoBehaviour
+{
+    [SerializeField] private CardResolutionZone resolutionZone;
+    [SerializeField] private GameEvent onEnemyTurnsConfirmed;
+
+    private List<Enemy> enemies;
```

Comment:
This isn't initialized anywhere. We get a NullRef when `ChooseCards` is invoked.